core.js

```diff
@@ -82,7 +83,7 @@ module.exports = function (xhr) {
       // For older servers, emulate JSON by encoding the request into an HTML-form.
       if (options.emulateJSON) {
           params.headers['content-type'] = 'application/x-www-form-urlencoded';
-          params.body = params.json ? {model: params.json} : {};
+          params.body = params.json ? {model: params.body || params.json} : {};
```

Comment:
Trying to think of the condition where params.json would be true, and there would be no params.body.  This would result in params.body being set to `{model: true}` which seems incorrect.

If `params.json` is `true`, it means new lines 70 and 71 ran above, which means `params.body` has to be set to either `options.attrs` or `model.toJSON(options)`. So at the bare minimum it's already set to `{}`;

Following this logic I believe that this line should read:

```
params.body = params.json ? { model: params.body } : {};
```